candybot/candybot/commands/admin/blacklist.py

```diff
@@ -4,18 +4,24 @@
 
 class BlacklistCommand(AdminCommand):
     name = "blacklist"
-    help = "Blacklists a user from interacting with CandyBot."
+    help = "Shows current blacklist or blacklists a user from interacting with CandyBot."
     aliases = ["bl"]
-    examples = ["@User", "User#1234", "123456789"]
+    examples = ["", "@User", "User#1234", "123456789"]
     argument_spec = ArgumentSpec([UserArgument], False)
     clean = True
     ignore = False
 
+    title = ":lock: CandyBot Blacklist"
```

Comment:
I dont think you can set this here because of how `self.send()` works.
Move it into the `if` on line 18.

Reply:
You will need to set it as `self.title` and not `title`. Ill need to fix `self.send()` at some point to make it dynamic instead of just looking at if a title is set :)